### ISSUE TITLE
[doc] calendar

### ISSUE THREAD

**Issue by darkiron** [brockpetrie/vue-moment#96]
How to use calendar filter, I want to obtain:
`mydate at this time`

I don't understand your explanations

**Comment by scherii** [comment]
The link provided in the readme should clarify what is meant by «calendar»:
http://momentjs.com/docs/#/displaying/calendar-time/

**Comment by darkiron** [comment]
nope add some info on moment function 
You can't call `moment().calendar()`
like:
```item.creation | moment().calendar()```
if I used this is great:
```item.creation | moment('calendar')```
but how defined parameters ? 
```{{ new Date() | moment('add', '6 days', 'calendar', null, { nextWeek: '[Happens in a week]' }) }}
<!-- "Happens in a week" -->````
what is the rule ?

**Comment by scherii** [comment]
I'd like to help you, but I don't understand your question.
Maybe [DeepL](https://www.deepl.com/translator) will help you help me help you? 😄

If I use the filter as described in the documentation, it works without problems.
```{{ new Date("July 21, 2018 10:40:03") | moment("calendar", "2018-07-20", { nextDay: "[Demain, à] H [h]"}) }}```

```{{ item.creation | moment("calendar", null, { # your options # }) }}```
`null` is the reference date, for `# your options #` see:

```
{
    sameDay: '[Today]',
    nextDay: '[Tomorrow]',
    nextWeek: 'dddd',
    lastDay: '[Yesterday]',
    lastWeek: '[Last] dddd',
    sameElse: 'DD/MM/YYYY'
}
```
